pkg/apis/core/validation/validation.go

```diff
@@ -5579,6 +5579,13 @@ func ValidateNamespace(namespace *core.Namespace) field.ErrorList {
 	for i := range namespace.Spec.Finalizers {
 		allErrs = append(allErrs, validateFinalizerName(string(namespace.Spec.Finalizers[i]), field.NewPath("spec", "finalizers"))...)
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.DefaultNamespaceLabelStamping) {
```

Comment:
Can we add test cases for this in `TestValidateNamespace` ?

Reply:
added three cases